src/main/java/io/cryostat/net/web/http/api/v1/TargetRecordingsPostHandler.java

```diff
@@ -173,10 +174,28 @@ public void handleAuthenticated(RoutingContext ctx) throws Exception {
                                         recordingOptionsBuilderFactory
                                                 .create(connection.getService())
                                                 .name(recordingName);
-                                boolean restart = false;
-                                if (attrs.contains("restart")) {
-                                    restart = Boolean.parseBoolean(attrs.get("restart"));
+
+                                String replace = attrs.get("replace");
+                                ReplacementPolicy replacementPolicy;
+                                if (StringUtils.isBlank(replace)) {
+                                    replacementPolicy =
+                                            ReplacementPolicy
+                                                    .NEVER; // Default to "never" if not provided
+                                } else {
+                                    switch (replace.toLowerCase()) {
+                                        case "always":
+                                            replacementPolicy = ReplacementPolicy.ALWAYS;
+                                            break;
+                                        case "stopped":
+                                            replacementPolicy = ReplacementPolicy.STOPPED;
+                                            break;
+                                        case "never":
+                                        default:
```

Comment:
This `default` would also handle the blank String cases except for `null` - maybe there's a way to refactor this to avoid the if-else + switch and do it with less nested logic?